scheduler/config.edn

```diff
@@ -43,7 +43,11 @@
        :keystore-path #config/env "COOK_KEYSTORE_PATH"
        :keystore-type "pkcs12"
        :keystore-pass "cookstore"}
- :rate-limit {:user-limit-per-m 1000000}
+ :rate-limit {:expire-minutes 120 ; Expire unused rate limit entries after 2 hours.
+              :job-submission {:bucket-size 15000
+                               :is-enforcing false
+                               :rate-per-minute 5000}
```

Comment:
Yes. Proposed name? Any reader that doesn't know what a TBF is shouldn't be touching this. If they know what a TBF is, there are only two configuration parameters and it would be obvious which is which.

Reply:
Maybe `:tokens-replenished-per-minute`?

Reply:
Ok.